### DISCUSSION TITLE
simple, incredible and complete

### DISCUSSION THREAD

**Discussion by davisilvarafacho** [mantinedev/mantine#6491] [Kind words]
I apologize if the words are a little confusing, I'm from Brazil and I'm using Google Translate to translate my sincere thanks to this incredible tool that is mantine.

I've never found a framework that suits me so much. The design really pleases me, the implementation is simple and it is full of features and behaviors that I love to develop.

For years I used bootstrap as my main framework, but after learning about mantine and implementing it in some personal projects and in the web project of the company I work for, I was able to see more closely how incredible the framework you developed is.

I find it almost impossible for any framework to take its place in my development and in my heart.

My sincere thanks for your incredible work.

**Comment by rtivital** [comment]
Thank you for kind words!

**Comment by abenhamdine** [comment]
I agree, Mantine is incredible ❤️
